pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/SubscriptionStats.java

```diff
@@ -118,15 +118,6 @@ public interface SubscriptionStats {
     /** Whether the Key_Shared subscription mode is AUTO_SPLIT or STICKY. */
     String getKeySharedMode();
 
-    /** This is for Key_Shared subscription to get the recentJoinedConsumers in the Key_Shared subscription. */
```

Comment:
Same: let's keep it until we remove old impl completely.

Reply:
https://github.com/apache/pulsar/pull/23352#discussion_r1788716055

Reply:
Addressed in #23424